core/app/models/spree/image/active_storage_attachment.rb

```diff
@@ -12,13 +12,8 @@ module Spree::Image::ActiveStorageAttachment
     validate :supported_content_type
 
     has_attachment :attachment,
-                   styles: {
-                   mini: '48x48>',
-                   small: '400x400>',
-                   product: '680x680>',
-                   large: '1200x1200>'
-                 },
-                 default_style: :product
+                   styles: Spree::Config.product_image_styles,
```

Comment:
I'm not sure about the naming here. I think this is a generic `image` attachment, so maybe the styles should be simply `Spree::Config.image_styles`. What do you think?

Reply:
Please disregard my comment above. I typed it but then I removed it, at least that's what I thought. While I think the naming for this class and the configurations may not be perfect, I think it's good enough.

Reply:
I had difficulties on it too, I felt I had to make a clear distinction from the taxon config and the product-image config